modules/video/include/opencv2/video/tracking.hpp

```diff
@@ -906,6 +906,8 @@ class CV_EXPORTS_W TrackerVit : public Tracker
         CV_PROP_RW std::string net;
         CV_PROP_RW int backend;
         CV_PROP_RW int target;
+        Scalar meanvalue{0.485, 0.456, 0.406};
+        Scalar stdvalue{0.229, 0.224, 0.225};
```

Comment:
This most probably affects Python and Java Bindings. Please define them as `CV_PROP_RW Scalar meanvalue` and move initialization to the constructor implementation.